src/Config.hs

```diff
@@ -1,21 +1,27 @@
 {-# LANGUAGE ApplicativeDo #-}
```

Comment:
Maybe move types from this module to `Parameters` to keep them with parsers. After that this module can probably be deleted

Reply:
Usually, I have a structure like 

```
src 
   | - NameOfThingToParse
             |- Types.hs
             |- Parser.hs
```
But you are right, this is an overkill for the task.